lib/Owner.rb

```diff
@@ -0,0 +1,73 @@
+module Bank
+  require "./account.rb"
+
+  class Owner
```

Comment:
It looks like you have more than one version of this class checked in to your PR. This one is in the `lib` folder and it looks like the other one is outside of the `lib` folder.